src/main/java/uk/gov/cshr/civilservant/service/OrganisationalUnitService.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.cshr.civilservant.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.cshr.civilservant.domain.OrganisationalUnit;
+import uk.gov.cshr.civilservant.repository.OrganisationalUnitRepository;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Service
+@Transactional
+public class OrganisationalUnitService {
+
+    private OrganisationalUnitRepository organisationalUnitRepository;
+
+    @Autowired
+    public OrganisationalUnitService(OrganisationalUnitRepository organisationalUnitRepository) {
+        this.organisationalUnitRepository = organisationalUnitRepository;
+    }
+
+    public ArrayList<OrganisationalUnit> getParentOrganisationalUnits() {
+        ArrayList<OrganisationalUnit> organisationalUnitArrayList = new ArrayList<>();
+
+        Iterable<OrganisationalUnit> organisationalUnits = organisationalUnitRepository.findAll();
```

Comment:
```suggestion
        organisationalUnitRepository.findAll().forEach(org....
```

Reply:
Does this not go against [Law Of Demeter](http://wiki.c2.com/?LawOfDemeter)?

Reply:
I would say that is more of a theoretical talk. 

If your chaining doesn't have any adverse effects - each link/function that has an input validates executes and outputs then you should take advantage of it to write less and achieve more.

This is where functional programming shines, makes the code for readable and succinct
+ saves us a great memory space for assigning a completely unnecessary collection of variables.